stripe/_http_client.py

```diff
@@ -316,14 +337,12 @@ def _request_with_retries_internal(
                     assert connection_error is not None
                     raise connection_error
 
-    def request(self, method, url, headers, post_data=None, *, _usage=None):
```

Comment:
These were mistakenly added during a bad merge of master -> beta, (merge conflict in the beta branch between adding "usage" and async refactor).